.gitignore

```diff
@@ -24,3 +24,5 @@ coverage
 build/
 dist/
 
+# env file
+.env
```

Comment:
it's good practice to end js files with a new line. for reference: [Newline necessary at the end of JavaScript files?](https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/#:~:text=So%2C%20to%20conclude%3A%20When%20in,you%20should%20put%20a%20newline.)